odl/set/domain.py

```diff
@@ -519,25 +513,17 @@ def insert(self, index, other):
         >>> rbox2 = IntervalProd([0, 0], [1, 0])
         >>> rbox.insert(1, rbox2)
         IntervalProd([-1.0, 0.0, 0.0, 2.0], [-0.5, 1.0, 0.0, 3.0])
-        >>> rbox.insert(2, [-1.0, 0.0])
-        IntervalProd([-1.0, 2.0, -1.0, 0.0], [-0.5, 3.0, -1.0, 0.0])
-
-        Can also insert by array
-
-        >>> rbox.insert(0, 1).squeeze() == rbox
-        True
-
-        See Also
-        --------
-        append
```

Comment:
I removed it because it failed, the functionality was not implemented (removed in some merge). Is it that useful? I can add it back, but it's slightly weird to call `IntervalProd` on the input, isn't it? In any case, it should be documented how to actually do this. A list of list approach should also work then.